docs/en/latest/plugins/request-validation.md

```diff
@@ -252,3 +254,30 @@ curl http://127.0.0.1:9080/apisix/admin/routes/5 -H 'X-API-KEY: edd1c9f034335f13
     }
 }
 ```
+
+**Custom reject message:**
```

Comment:
What about giving a request/response example for the custom rejected message?

Reply:
Good suggestion.

Reply:
How about deal with it in another PR?